src/scss/_themes.scss

```diff
@@ -105,7 +105,8 @@ body.theme-light {
     border-bottom: 1px solid var(--color-bg-dark);
   }
 
-  .step-error {
+  .step-error,
+  .step-skipped {
```

Comment:
Forgive me inexperience with this 🐼 

Looking at this, it almost suggests to me that the `.step-skipped` class will apply the color red?

Am I reading or understanding this correctly?

Reply:
no worries. good question. yes, for the text color. we could keep it white/gray 🤔

Reply:
Ahhhhhh this would be for the text color in the log output 👍 

For some reason when I was reading this it lead me to believe that color would be applied across the board 😛

Reply:
i think i'm ok with the "warning" text color for now, but we discuss further beyond this PR 👍